Extensions/ArtifactEngine/Store/artifactItemStore.ts

```diff
@@ -30,7 +30,7 @@ export class ArtifactItemStore {
 
     public itemsPendingProcessing(): boolean {
         var processingTickets = this._downloadTickets.filter(x => x.state === models.TicketState.Processing || x.state === models.TicketState.InQueue);
-        return processingTickets ? processingTickets.length != 0 : false;
+        return processingTickets.length != 0;
```

Comment:
wasn't the previous code safer?

Reply:
The code was dead code, filter never returns undefined. It was caught using codecoverage.